README.md

```diff
@@ -497,6 +497,90 @@ if (html instanceof Promise) {
 
 <br />
 
+### Why there is no `context` API?
+
+We choose to not provide a `context` API. Here the reasons why:
+
+This library only outputs strings (or Promises that resolve to strings), we don't manage
+lifecycle or state. So the main selling point to having a `context` API is to avoid prop
```

Comment:
```suggestion
lifecycle or state. So the main selling point to having a `context` API is to avoid [prop drilling](https://www.geeksforgeeks.org/what-is-prop-drilling-and-how-to-avoid-it).
```

Reply:
unusual term, a reference is good